### PR TITLE
[EN] Immutant2 migration

### PR DESCRIPTION
**Pull request by ericnormand**
https://www.pivotaltracker.com/story/show/87464434

To run the tests:

lein with-profile test immutant test -j path/to/wildfly

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -10,18 +10,23 @@
                  [turbovote.datomic-toolbox "0.2.5-SNAPSHOT" :exclusions [com.datomic/datomic-pro]]
                  [com.datomic/datomic-pro "0.9.5130" :exclusions [org.slf4j/slf4j-nop]]
                  [clj-aws-s3 "0.3.8"]
+                 [clj-time "0.9.0"]
                  [democracyworks.squishy "1.0.0"]
                  [org.clojure/data.csv "0.1.2"]
-                 [turbovote.imbarcode "0.1.4-SNAPSHOT"]
+                 [turbovote.imbarcode "0.1.4-SNAPSHOT"
```
Comment (cap10morgan) [inline project.clj]: We should really not depend on snapshots. Does this actually need unreleased functionality and can we release it if so? @tie-rack?

**Comment by cap10morgan** [comment]
Other than snapshot deps (which we can clean up later), this LGTM.

File: project.clj (hunk 1 above)
Comment (tie-rack) [inline project.clj]: There is a 0.1.5 release that will have whatever it is this was reliant on.

Comment (cap10morgan) [inline project.clj]: Great! Then let's change this from `0.1.4-SNAPSHOT` to `0.1.5`.

**Pull request merged**